tests/appsec/waf/test_telemetry.py

```diff
@@ -224,6 +224,9 @@ def _validate_headers(headers, request_type):
     if context.library == "python":
         # APM Python migrates Telemetry to V2
         expected_headers["DD-Telemetry-API-Version"] = "v2"
+    elif context.library == "nodejs":
```

Comment:
We still test the prod version:

```suggestion
    elif context.library > "nodejs@4.20.0":
```